materials/module-1/worksheets/2-peer-to-peer-file-sharing-on-a-mesh.md

```diff
@@ -0,0 +1,80 @@
+Peer-to-peer file sharing on a mesh
+===================================
+
+## Connecting to your Raspberry Pi
+
+1. Verify that the Raspberry Pi has its WiFi adapter(s) proper connected SD card is in place. Power on the device and wait for a solid green LED with a flashing red LED.
+
+1. Connect your computer's WiFi Client to the Raspberry Pi's WiFi Access Point:
+
+        SSID:       <hostname>
+        Password:   password
+
+1. Once connected to the Access Point from your computer, you can connect to it with SSH password `root`:
+
+        $ ssh root@<hostname>.local
+
+    For example, on `bloor` the command would be `ssh root@bloor.local`. You can optionally connect to the IP address directly without using mDNS `ssh root@10.0.0.1`.
+
+1. Find a Raspberry Pi near you can try to ping it to verify the mesh network interface is working. For example:
+
+        root@bloor:~# ping college.local
+
+    Hit `Ctrl + C` to stop the ping.
+
+You are now a mesh node operator running commands from your pet Raspberry Pi, which is connected to the rest of the mesh network we just formed in this room! For the rest of this activity, we will continue to use `bloor` to represent your node, and `college` to represent another node near you. Be sure to replace these with the name of your own pet Raspberry Pi. 
+
+## Running the InterPlanetary File System (IPFS)
+
+1. Start IPFS in the background:
+
+        root@bloor:~# docker load --input ~/docker/tomeshnet-ipfs-0.1.tar
+        root@bloor:~# docker run --name ipfs --network host --detach tomeshnet/ipfs:0.1 
```

Comment:
This formatting is non intuitive with the line breaks, would add `$` in front of each line, and set that expectation in the class (could consider live demoing opening a terminal and spending 5 mins pointing stuff out?)

Reply:
Need to seperate prompt
```root@bloor:~# ```
from what to type in
```docker load --input ~/docker/tomeshnet-ipfs-0.1.tar```

Its confusing to some people who don't know the difference

Also for line break issues just double space the different commands so when it breaks its obvious it still part of the same command

Reply:
>  would add $ in front of each line

I see the reason ben didnt just do "$" and leave it at that. But perhaps change bloor to `mynode` or `examplenode` or something